Hyper/Camera.cs

```diff
@@ -41,6 +46,10 @@ public Camera(Vector3 position, float aspectRatio)
 
         public Vector3 Right => _right;
```

Comment:
I don't like these. No reason to have both RIght and _right when we can just have Right {get; private set;}